doc/devel/coding_guide.rst

```diff
@@ -165,6 +165,48 @@ C/C++ extensions
 
 
 
+PR Review guidelines
+====================
+
+* We run on a 'use your judgement' system (for better or worse) and do
+  not have a strict nose-count rule.  If you have a commit bit, then
```

Comment:
@tacaswell I'd switch to a 2 core dev validation, with explicit approval for merge except for documentation changes.
Else, we'll stay in the situation we are in, where everyone comments and no one merges.